integration-tests/prepare.sh

```diff
@@ -58,6 +58,14 @@ function init_tendermint() {
     if [ ! -z "${CI}" ]; then
         sudo chmod -R 777 ./tendermint
     fi
+
+    index_all_tags "tendermint"
+}
+
+# @argument Tendermint directory
+function index_all_tags() {
+    print_step "Disabling empty blocks for ${1}"
```

Comment:
```suggestion
    print_step "Enable tag indexing for ${1}"
```

Reply:
> is tag indexing required for integration test?

Not at the moment. I use it to test between other projects I am working on. So I enable it to facilities testing and development.